db/query_more_test.go

```diff
@@ -94,7 +94,7 @@ var (
 
```

Comment:
Test ran in parallel, but `createCollectionWithData` was modifying the global variable!
Now that method returns the modified dataset from the sample data, so parallel runs use their own data sample.

Reply:
😨